project_stage_closed/migrations/2.0.0/post-remove-project-task-type-closed-field.py

```diff
@@ -0,0 +1,16 @@
+import logging
```

Comment:
This should be done on pre, not on post, as on post the column is already dropped by Odoo update mechanism.